src/components/panels/ConfigPanel/TextAreaConfig.tsx

```diff
@@ -51,6 +74,10 @@ export default function TextAreaConfig({
     setNewValue(config.name, config.default);
   }
 
+  function handleShowDescription() {
+    setShowDescription(!showDescription);
```

Comment:
These set of words are confusing 😂

Reply:
I know, I feel like I need to add a comment that says `description` just for fun 😆